Date:	Thu, 04 Aug 2005 19:31:18 -0700
From: Alex Aizman <itn780@yahoo.com>
To: James Bottomley <James.Bottomley@SteelEye.com>
Cc: "David S. Miller" <davem@davemloft.net>,
	SCSI Mailing List <linux-scsi@vger.kernel.org>,
	Christoph Hellwig <hch@lst.de>
Subject: [ANNOUNCE 4/7] open-iscsi/linux-iscsi-5 Initiator: open-iscsi-netlink.patch

open-iscsi-netlink.patch - include/linux/netlink.h changes (added NETLINK_ISCSI).

Signed-off-by: Alex Aizman <itn780@yahoo.com>
Signed-off-by: Dmitry Yusupov <dmitry_yus@yahoo.com>
Signed-off-by: Mike Christie <michaelc@cs.wisc.edu>
Acked-by: Hannes Reinecke <hare@suse.de>

diff --git a/include/linux/netlink.h b/include/linux/netlink.h
--- a/include/linux/netlink.h
+++ b/include/linux/netlink.h
@@ -22,6 +22,7 @@
 #define NETLINK_KOBJECT_UEVENT	15	/* Kernel messages to userspace */
 #define NETLINK_TAPBASE		16	/* 16 to 31 are ethertap */
 
+#define NETLINK_ISCSI		31	/* iSCSI Open Interface */
 #define MAX_LINKS 32		
 
 struct sockaddr_nl


